src/scheduling.jl

```diff
@@ -1,4 +1,5 @@
 using MetaGraphs
+using OhMyThreads: @tasks, @set
```

Comment:
```suggestion
using OhMyThreads: @tasks, @set
using ThreadPinning: pinthreads
pinthreads(:cores)
```

maybe you also want to do these

Reply:
Nope, we tried `ThreadPinning` and decided against it - we prefer to use standard tooling outside of Julia